tests/code-splitting/subtests/dynamic-import-preloading/index.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Dynamic import Preloading
+importance: 1
+shortDesc: 'Can code splitting be preloaded?'
+---
+
+# Introduction
+
+While loading the module which has been code spllited, it is important to load its dependencies in parallel to
+optimize the load time.
+
+# The Test
+
+This test checks to see if it's possible to start network request for the dependecies as soon as the code splitted module start loading.
+**index.js**
```

Comment:
```suggestion
This test checks to see if it's possible to request code-splitted dependency bundles before the consuming bundle has been loaded, either via [preload](https://web.dev/preload-critical-assets/) or async script loading.

**index.js**
```

Reply:
I think you both talk about different tests that both make sense.

Preloading/Prefetching: Load files related to a `import()` before they are really requested.

Parallel loading of dependencies: Does an `import()` cause multiple parallel requests or an waterfall of requests?